### PR TITLE
15 update readme language ext webpart

### PR DESCRIPTION
**Pull request by RVany**
Readme upgraded in new template.

### CODE REVIEW

**Review by Steph4104 — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -1,18 +1,27 @@
-# spfx-extend-lang
+# Extend Language Extension
 
 ## Summary
 
 Extend language select to add link to account language settings.
 
-Language toggles on Hub sites will be automatically be targeted by this extension. To target sub-sites of a hub site that also has language toggles, change the siteIds values in the extension properties.
+Language toggles on Hub sites will be automatically be targeted by this extension. To target sub-sites of a hub site that also has language toggles, change the siteIds values in the extension properties. This extension is intended to be deployed tenant wide. When a user visit first time to the site, a tour about the extension will be displayed.
```
[the remark this answers is not in the document]
Comment (RVany) [reply README.md:7]: May I know when it will trigger?

File: README.md [hunk 2]
```diff
@@ -1,18 +1,27 @@
...
+![Preview of Tour](sharepoint/assets/lng-ext-tour2.png)
+![Preview of Tour](sharepoint/assets/lng-ext-tour3.png)
 
-## Used SharePoint Framework Version
-
-![version](https://img.shields.io/badge/version-1.11-green.svg)
+## Prerequisites
+None
+## API permission
+Microsoft Graph - User.ReadBasic.All
```
[the remark this answers is not in the document]
Comment (RVany) [reply README.md:21]: I think in onit() method of **_ExtendLanguageApplicationCustomizer.ts_**.  I believe it used to create the url for the language & setting.  We need to direct to profile language setting of the specific user(current user) right?

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Steph4104) [reply README.md:7]: In tour.ts line 282 you can see that its looking for gcxLangTour + the language value

[the remark this answers is not in the document]
Comment (RVany) [reply README.md:7]: Ok. But when and how that url parameter going to trigger.  I thought the registration app redirecting to the url with the parameter gcxLangTour + the language value.

[the remark this answers is not in the document]
Comment (RVany) [reply README.md]: I changed the sentence as "When the ***gcxLangTour*** parameter triggers in the URL, a tour about the extension will be displayed." Is it correct?

[the remark this answers is not in the document]
Comment (RVany) [reply README.md]: Modified with above text.

**Review by Steph4104 — APPROVED** [review]
Make sure to review what you copy past

**Pull request merged**